src/components/Tabs/index.js

```diff
@@ -88,17 +89,21 @@ const Tab = styled.div`
 const checkIfOverflows = ({ offsetWidth = 0, scrollWidth = 0 } = {}) =>
   offsetWidth < scrollWidth;
 
-const TabItemButton = styled.div.attrs(props => {
+const TabItemButton = styled.button.attrs(props => {
   const { isActive, dataIndex, ...rest } = props;
   return {
     className: isActive ? "tab__button--active" : "",
+    role: "tab",
     "data-index": dataIndex,
+    "aria-selected": isActive,
     ...rest
   };
 })`
-  display: block;
   padding: ${padding} 0 ${spacing.cozy} 0;
   cursor: pointer;
+  background-color: ${themes.global.transparent};
+  border: none;
```

Comment:
you might also want to add `appearance: none` to reset default styles in safari

Reply:
Will do. Thank you